scripts/setup_linux.sh

```diff
@@ -11,8 +11,23 @@ hash conda 2>/dev/null || {
 echo "Installing system dependencies"
 echo "You will probably be asked for your sudo password."
 sudo apt-get update
-sudo apt-get install -y python-pip python-dev swig cmake build-essential \
-  zlib1g-dev libopenmpi-dev
+sudo apt-get install -y \
+  python-pip \
```

Comment:
python-pip is absolutely not necessary, because we should only be installing this inside the conda environment